pkg/kudoctl/packages/resolver/resolver.go

```diff
@@ -0,0 +1,72 @@
+package resolver
+
+import (
+	"fmt"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/http"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/repo"
+)
+
+// Resolver is a packages resolver and is any implementation which can find/discover a packages.
+// Even Repos are finders. Local and URL Finders current do nothing with the version information.
```

Comment:
please update the docs here, I was kind of confused to read about finders while that's another name for the same thing I feel. Should we rename Finders to Resolvers?